.scalafmt.conf

```diff
@@ -1,6 +1,6 @@
 version = "2.3.2"
 maxColumn = 100
-
+lineEndings=preserve
```

Comment:
hm, can't we keep the `unix` line ending?

Reply:
We tried, it actually reformat everything